src/stylesheets/components/_alerts.scss

```diff
@@ -3,40 +3,40 @@ $usa-custom-alerts: () !default;
 $usa-custom-alerts-bar: () !default;
 
 $usa-alerts: (
-  success: $color-green-lightest,
-  warning: $color-gold-lightest,
-  error: $color-secondary-lightest,
-  info: $color-primary-alt-lightest
+  success: color('success-lighter'),
+  warning: color('warning-lighter'),
+  error: color('error-lighter'),
+  info: color('accent-cool-lighter')
 );
 
 $usa-alerts-bar: (
-  success: $color-green,
-  warning: $color-gold,
-  error: $color-secondary,
-  info: $color-primary-alt
+  success: color('success'),
+  warning: color('warning'),
+  error: color('error'),
+  info: color('accent-cool')
```

Comment:
I think I will add `info` to the state palette. OK to just do it in this branch?

Reply:
sure thing!